scripts/integration-tests/prometheus/prometheus-integration-test.sh

```diff
@@ -5,9 +5,16 @@ set -xe
 rm -rf /tmp/m3dbdata/
 mkdir -p /tmp/m3dbdata/
 
-echo "Build docker images"
+PARAM_TEST_BUILD="${TEST_BUILD:-true}"
```

Comment:
Are these injected by semaphore? Or just more meant for local testing?

Reply:
Local testing yeah.